.github/workflows/test-docker-image-build.yml

```diff
@@ -0,0 +1,23 @@
+# https://docs.github.com/en/actions/learn-github-actions/understanding-github-actions
+
+name: Test Docker image build
+
+on:
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - name: Build Docker image
+      run: |
+        $ImageTag = "${{ github.sha }}"
+        $RepoName = "public/quantum/samples"
+
+        $LocalTag = "${RepoName}:${ImageTag}"
+        $RemoteRepo = "${{ secrets.ACR_REGISTRY }}/${RepoName}"
```

Comment:
Where does this variable get used? Does it get used in a different context after this job?

Reply:
It is a copy-paste problem, good find.

It was used in the fragment where I copied this from. I will remove this var.

Reply:
Removed.